src/quo2/components/community/community_view.cljs

```diff
@@ -1,16 +1,20 @@
 (ns quo2.components.community.community-view
-  (:require [quo2.components.community.style :as style]
+  (:require [quo.gesture-handler :as gesture-handler]
```

Comment:
hmm, you should be using `react-native.gesture` instead

Reply:
oh absolutely @J-Son89. This one slipped from the old code.